public/components/getting_started/components/getting_started.tsx

```diff
@@ -24,7 +24,7 @@ export const NewGettingStarted = (props: ExtendedHomeProps) => {
   useEffect(() => {
     chrome.setBreadcrumbs([
       {
-        text: 'Getting Started',
+        text: 'Get started',
```

Comment:
should this just use `observabilityGettingStartedTitle`?

Reply:
Yes, thank you.